flintrock/flintrock.py

```diff
@@ -226,6 +226,8 @@ def cli(cli_context, config, provider):
 @click.option('--ec2-ebs-optimized/--no-ec2-ebs-optimized', default=False)
 @click.option('--ec2-instance-initiated-shutdown-behavior', default='stop',
               type=click.Choice(['stop', 'terminate']))
+@click.option('--ec2-user-data', default='',
```

Comment:
I changed `--ec2-user-data` option type to Click's `File` type, and remove `_get_user_data()`.  
I think Click's `File` type is more appropriate than Click's `Path` type since user-data script should be opened and read. 
Could you give me a comment for this?